.travis.yml

```diff
@@ -37,7 +37,7 @@ matrix:
   include:
     - name: emscripten
       os: linux
-      env: STATIC_LIBS=ON EMSCRIPTEN=ON
+      env: STATIC_LIBS=ON EMSCRIPTEN=ON ALTERNATIVE_CMAKE=3.9.3-Linux-x86_64
```

Comment:
I assume this is because you can now remove the `Install modern CMake` section?

Reply:
Yes, both emscripten and cmake-2.8.12 builders need a different cmake version, so they both use the same mechanism